composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "lukepolo/laracart",
+  "name": "tyloo/laracart",
```

Comment:
It's just to be able to use my changes on my current project. Ofc I'll set it back to lukepolo ;)